src/OpenLoco/Window.h

```diff
@@ -83,102 +62,15 @@ namespace OpenLoco::Ui
-        out.content |= (1 << 29);
-
-        return out;
-    }
-
-#define makeDropdownWidgets(...) \
-    makeWidget(__VA_ARGS__),     \
-        makeDropdownButtonWidget(__VA_ARGS__)
-
-    [[maybe_unused]] static constexpr widget_t makeDropdownButtonWidget(Gfx::point_t origin, Gfx::ui_size_t size, [[maybe_unused]] widget_type type, uint8_t colour, [[maybe_unused]] uint32_t content = 0xFFFFFFFF, [[maybe_unused]] string_id tooltip = StringIds::null)
-    {
-        const int16_t xPos = origin.x + size.width - 12;
-        const int16_t yPos = origin.y + 1;
-        const uint16_t width = 11;
-        const uint16_t height = 10;
-
-        return makeWidget({ xPos, yPos }, { width, height }, widget_type::wt_11, colour, StringIds::dropdown);
-    }
-
-#define makeStepperWidgets(...)                 \
-    makeWidget(__VA_ARGS__),                    \
-        makeStepperDecreaseWidget(__VA_ARGS__), \
-        makeStepperIncreaseWidget(__VA_ARGS__)
-
-    [[maybe_unused]] static constexpr widget_t makeStepperDecreaseWidget(Gfx::point_t origin, Gfx::ui_size_t size, [[maybe_unused]] widget_type type, uint8_t colour, [[maybe_unused]] uint32_t content = 0xFFFFFFFF, [[maybe_unused]] string_id tooltip = StringIds::null)
-    {
-        const int16_t xPos = origin.x + size.width - 26;
-        const int16_t yPos = origin.y + 1;
-        const uint16_t width = 13;
-        const uint16_t height = size.height - 2;
-
-        return makeWidget({ xPos, yPos }, { width, height }, widget_type::wt_11, colour, StringIds::stepper_minus);
-    }
-
-    [[maybe_unused]] static constexpr widget_t makeStepperIncreaseWidget(Gfx::point_t origin, Gfx::ui_size_t size, [[maybe_unused]] widget_type type, uint8_t colour, [[maybe_unused]] uint32_t content = 0xFFFFFFFF, [[maybe_unused]] string_id tooltip = StringIds::null)
-    {
-        const int16_t xPos = origin.x + size.width - 13;
-        const int16_t yPos = origin.y + 1;
-        const uint16_t width = 12;
-        const uint16_t height = size.height - 2;
-
-        return makeWidget({ xPos, yPos }, { width, height }, widget_type::wt_11, colour, StringIds::stepper_plus);
-    }
-
-    constexpr widget_t makeTextWidget(Gfx::point_t origin, Gfx::ui_size_t size, widget_type type, uint8_t colour, string_id content, string_id tooltip = StringIds::null)
-    {
-        widget_t out = {};
-        out.left = origin.x;
-        out.right = origin.x + size.width - 1;
-        out.top = origin.y;
-        out.bottom = origin.y + size.height - 1;
-        out.type = type;
-        out.colour = colour;
-        out.text = content;
-        out.tooltip = tooltip;
-
-        return out;
-    }
-
-    constexpr widget_t widgetEnd()
-    {
-        widget_t out = {};
-        out.type = widget_type::end;
-
-        return out;
+        constexpr uint8_t none = 0;
+        constexpr uint8_t horizontal = (1 << 0);
+        constexpr uint8_t vertical = (1 << 1);
+        constexpr uint8_t both = (1 << 0) | (1 << 1);
```

Comment:
Now that this is a namespace, we could just write:
```suggestion
        constexpr uint8_t both = horizontal | vertical;
```